packages/core/src/sizing/allSizes.ts

```diff
@@ -7,12 +7,4 @@
 
 import { Size } from './Size';
 
-export const allSizes: Size[] = [
-  Size.XXS,
-  Size.XS,
-  Size.S,
-  Size.M,
-  Size.L,
-  Size.XL,
-  Size.XXL,
-];
+export const allSizes: Size[] = Object.values(Size)
```

Comment:
Would you mind adding a semicolon and breaking line here (ending file with an empty line)? I still have to set up a pre-commit git hook to run prettier.